frontend/packages/yki/jest-environment-patched-jsdom.js

```diff
@@ -0,0 +1,13 @@
+// test.environment.js
+
+const JSDOMEnvironment = require("jest-environment-jsdom").default;
+
+module.exports = class CustomTestEnvironment extends JSDOMEnvironment {
+    async setup() {
+        await super.setup();
+        this.global.TextEncoder = TextEncoder;
+        this.global.TextDecoder = TextDecoder;
+        this.global.Response = Response;
+        this.global.Request = Request;
+    }
+};
```

Comment:
Mihin nämä uudet `jest-environment-patched-jsdom.js`-tiedostot liittyvät?

Reply:
Tää liittyi siihen, että jsdom päivityksessä nuo globaalit oli poistunu mutta jokin riippuvuus ykissä tarvi niitä vielä. En nyt muista mikä kirjasto se oli. Taisi olla tämä minkä kautta löysin tän ratkaisun: https://stackoverflow.com/questions/68468203/why-am-i-getting-textencoder-is-not-defined-in-jest

Lisäsin myös tähän tiedostoon kommentiksi tämän.